src/apps/relay/dbdrivers/dbd_redis.c

```diff
@@ -268,7 +268,7 @@ redis_context_handle get_redis_async_connection(struct event_base *base, redis_s
         }
       }
 
-      ret = redisLibeventAttach(base, co->host, co->port, co->password, atoi(co->dbname));
+      ret = redisLibeventAttach(base, co->host, (uint16_t)co->port, co->password, atoi(co->dbname));
```

Comment:
Pls review if you can change the type to uint16_t in the struct rather than adding cast